### DISCUSSION TITLE
Multiple font weight support

### DISCUSSION THREAD

**Discussion by Joshua-Flores** [trussworks/react-uswds#2099] [Q&A]
I noticed that only 3 font weights are supported for source sans pro web which is embedded by default. Are there any plans to expand this to more than that?

**Comment by haworku** [comment]
@Joshua-Flores can you expand a little more on what you are looking for here and what the current pain is. Which file are you referring to or which component is limiting you from adjusting font weights as you would like? 

 I'm assuming you are using uswds directly in your application as well as our component library  (also let me know if that's incorrect, using uswds directly along the lib is our recommended approach ).

**Comment by werdnanoslen** [comment]
By default, [USWDS makes only those three weights available](https://designsystem.digital.gov/documentation/settings/#typography-settings) by default without changing certain variables. In your project, you can [customize those variables](https://github.com/trussworks/react-uswds/blob/main/docs/styles_and_assets.md#configuring-uswds-scss-variables) in an scss file like this:
```
// your-project/_uswds-theme.scss

@use 'uswds-core' with (
  $theme-font-weight-thin: 200,
  $theme-font-weight-medium: 500,
  $theme-font-weight-semibold: 600,
  $theme-font-weight-heavy: 800,

  // or, as a shortcut for all of the above
  $theme-generate-all-weights: true
);
```
